### PR TITLE
Solves #347

### PR DESCRIPTION
**Pull request by benoror**
solves Issue #347

### CODE REVIEW

**Comment by kentcdodds** [comment]
Do you think that you could add a test that reproduces the original bug and shows that this fixes that bug? TDD style :-)

**Comment by benoror** [comment]
I can do that!

**Comment by benoror** [comment]
Done, I'm not sure if that's the proper place to test it

File: src/directives/formly-field.test.js [hunk 1]
```diff
@@ -997,6 +997,28 @@ describe('formly-field', function() {
       expect(() => compileAndDigest()).to.throw();
     });
 
+    it(`should share the form with a fieldGroup`, () => {
```
Comment (kentcdodds) [inline src/directives/formly-field.test.js]: I think you put this in the `describe` for `model`. Do you mind creating a new `describe('fieldGroup')` and putting this test in that one? While you're at it, you could move the `describe(fieldGroup with specified "key" property)` inside that new one if you want. :+1:

**Comment by kentcdodds** [comment]
Test looks solid. Just let me know about what you think about my other comment.

**Comment by benoror** [comment]
Great, let me rewrite it for a better organization

**Comment by kentcdodds** [comment]
Can't believe how long that file's getting. I may consider breaking it up a little bit eventually. Haha.

**Comment by benoror** [comment]
ready!

**Comment by kentcdodds** [comment]
Next time we see each other, allow me to give you a hi five!

highest-of-fives-explosion

**Comment by benoror** [comment]
no prob, glad to be helpful! :hand:

**Comment by kentcdodds** [comment]
This has been released in `6.13.1`. Thanks again.

**Pull request merged**